src/main/resources/templates/fragments/header.html

```diff
@@ -165,4 +165,4 @@
   </header>
 </th:block>
 </body>
-</html>
+</html>
```

Comment:
Were the above changes done by you? Seems to be a lot of unrelated changes, is this branch up to date with master?

Reply:
I did a reverse merge with master and then reformatted before pushing. I'm a bit confused because any changes that have crept in from the master merge shouldn't be causing disparities with master. The reformatting may have changed some....